studio/components/interfaces/Auth/SmtpForm/SmtpForm.tsx

```diff
@@ -244,6 +244,15 @@ const SmtpForm = () => {
                     labelOptional="In seconds"
                     descriptionText="How long between each email can a new email be sent via your SMTP server."
                   />
+                  <Input
+                    type="number"
+                    name="RATE_LIMIT_EMAIL_SENT"
+                    id="RATE_LIMIT_EMAIL_SENT"
+                    min={0}
+                    label="Rate limit for sending emails"
+                    descriptionText="How many emails can be sent per hour."
+                    actions={<span className="text-scale-900 mr-3">per hour</span>}
```

Comment:
could we shift this to `labelOptional="Per hour"` - for consistency and also cause the input fields vanilla increment/decrement button overlays this text

Reply:
shall i just fix it in UI?

Reply:
the default arrows need to be removed actually

Reply:
done